pkg/meta/labels.go

```diff
@@ -34,38 +99,21 @@ const (
 	// to be removed by a pending node removal.
 	ClientRoutingLabel = "vertica.com/client-routing"
 	ClientRoutingVal   = "true"
+	//
+	// This is set in the pods, and is used by statefulset as a pod selector. This
+	// stays constant for the life of the statefulset, and is unique across
+	// statefulsets for other subclusters in the database. It may appear like it
+	// is derived from a subcluster name, but is does not get updated for
+	// subcluster rename. So, it shouldn't be treated as such.
+	SubclusterSelectorLabel = "vertica.com/subcluster-selector-name"
```

Comment:
Does that mean the sts name no longer changes when the subcluster is renamed?

Reply:
Yes, this will stay static. This is tied to the statefulset name, which we'll keep static for the rename too.